bundles/binding/org.openhab.binding.mochadx10/src/main/java/org/openhab/binding/mochadx10/commands/MochadX10DimCommand.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mochadx10.commands;
+
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.library.types.PercentType;
+
+/**
+ * Implementation of the X10 Dim command
+ * 
+ * @author Jack Sleuters
+ * @since  1.7.0
+ *
+ */
+public class MochadX10DimCommand extends MochadX10Command {
```

Comment:
I would like to have the difference between the Dim and Bright command in the different classes. Else, I would introduce some property which would state whether the comman is dim not bright, less OO in my opinion.